src/include/btmem.h

```diff
@@ -246,6 +246,17 @@ struct __wt_save_upd {
     bool restore; /* Whether to restore this saved update chain */
 };
 
+/*
+ * WT_DELETE_HS_UPD --
+ *	Update needs to be deleted from the history store.
```

Comment:
I would say "Update **that** needs ...". Or "Information related to an update that should be removed from the history store".